wire-ios-request-strategy/Sources/APIs/MessageAPI.swift

```diff
@@ -265,6 +271,87 @@ class MessageAPIV4: MessageAPIV3 {
     override var apiVersion: APIVersion {
         .v4
     }
+
+    private let protobufContentType = "application/x-protobuf"
+
+    override func broadcastProteusMessage(message: any ProteusMessage) async throws -> (Payload.MessageSendingStatus, ZMTransportResponse) {
```

Comment:
Will these overriden methods be tested in tests?

Reply:
There are no existing tests for `MessageAPI`, and I'm thinking that since it will soon be replaced in the refactor then perhaps it's not worth writing a whole test suite for the object. But I'm also happy to in a separate PR if you think it's worth it.

Reply:
I was thinking we could just copy and paste and change API versions so give us some temporary coverage, but if there aren't existing tests already then the overhead is probably to high. 

I would say it would only be worth doing it if those tests can be written in a way that they don't break with the refactor. That way they can give some protection when integrating the new code. If this is not the case then I don't think it is worth the effort. Feel free to close this comment.

Reply:
With the refactor we'll be rewriting this whole API and ensuring it has comprehensive tests.